build/void.txt

```diff
@@ -0,0 +1,30 @@
+account
+attachable
+bill
+billPayment
+class
+companyInfo
+creditMemo
+customer
+department
+employee
+estimate
+exchangerate
+invoice
+item
+journalEntry
+payment
+paymentMethod
+preferences
+purchase
+purchaseOrder
+refundReceipt
+salesReceipt
+taxAgency
+taxCode
+taxRate
+taxService
+term
+timeActivity
+vendor
+vendorCredit
```

Comment:
So the `build` directory was only ever used the first time I created the library. I've never run the "build" script again, which was just an easy way to generate a couple hundred CRUD methods. But I didn't want to throw it away, in case I ever needed it again. A little confusing for folks coming to this library for the first time. Sorry about that. But you don't really need this file.

Reply:
I figured. I just wanted to follow the pattern from the others. We can ignore this.